docs/pages/store/reference/store-core.mdx

```diff
@@ -774,16 +774,6 @@ event Store_SetRecord(
 );
 ```
 
-**Parameters**
-
-| Name             | Type            | Description                                             |
-| ---------------- | --------------- | ------------------------------------------------------- |
-| `tableId`        | `ResourceId`    | The ID of the table where the record is set.            |
-| `keyTuple`       | `bytes32[]`     | An array representing the composite key for the record. |
-| `staticData`     | `bytes`         | The static data of the record.                          |
-| `encodedLengths` | `PackedCounter` | The encoded lengths of the dynamic data of the record.  |
-| `dynamicData`    | `bytes`         | The dynamic data of the record.                         |
-
```

Comment:
these docs changes seem unintended, or am i missing something?

Reply:
possible that the docs are out of date, but let's ignore this change for this PR and move these changes to a separate PR